resources/lib/kodiutils.py

```diff
@@ -9,6 +9,15 @@
 import xbmcplugin
 from statichelper import from_unicode, to_unicode
 
+
+class StubException(Exception):
```

Comment:
I would rather define it like this:
```python
        try:  # Python 3
            from json import loads, JSONDecodeError
            class ValueError(Error):
                pass
         except ImportError:  # Python 2
            from json import loads
            class JSONDecodeError(Error):
                pass
```

Reply:
BTW I think ValueError also exists in Python 3, so I don't think we have to define it here.